Felienne/hedy
# Switch Babel dependency to YAML file for tutorials

**Description**
In this PR we switch the tutorial translations from Babel to a YAML file to better enable us to add new levels and steps without losing the overview. We cache all tutorial language / keyword language combinations on server start to reduce server load and enable dynamic keywords. Currently this is not relevant, but probably will be when adding the tutorials for new levels.

**Fixes**
This PR fixes #3165.

**How to test**
As this is a re-structuring task all should work as before. To verify log in and navigate to `/tutorial`, verify that all steps stil work as expected.
